src/Com.RusticiSoftware.Cloud.V2/Com.RusticiSoftware.Cloud.V2.nuspec

```diff
@@ -15,28 +15,28 @@
              users to earily find other packages by the same owners.  -->
         <owners>rustici-software-systems</owners>
 
-        <license type="extention">Apache-2.0</license>
+        <license type="expression">Apache-2.0</license>
 
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <developmentDependency>false</developmentDependency>
 
         <!-- The description can be used in package manager UI. Note that the
              nuget.org gallery uses information you add in the portal. -->
         <description>Swagger Generated C# Client for SCORM Cloud API v2</description>
-            <copyright>https://cloud.scorm.com/sc/scorm_cloud_terms_of_service.html</copyright>
+        <copyright>https://cloud.scorm.com/sc/scorm_cloud_terms_of_service.html</copyright>
 
         <tags>SCORM rustici elearning AICC xAPI</tags>
         <!-- Dependencies are automatically installed when the package is installed -->
         <dependencies>
 
             <dependency id="NewtonSoft.Json" version="10.0.3" />
             <dependency id="JsonSubTypes" version="1.2.0" />
-            <dependency id="RestSharp" version="105.1.0" />
+            <dependency id="RestSharp" version="106.0.1" />
 
         </dependencies>
     </metadata>
     <files>
         <!-- A readme.txt will be displayed when the package is installed -->
         <file src="..\..\README.md" target="" />
     </files>
```

Comment:
It seems the `<files>`-Tag is missing items for the library itself. See e.g. https://docs.microsoft.com/en-us/nuget/reference/nuspec#including-assembly-files

Reply:
Sorry about that. Will be fixing and creating a new version shortly.

Reply:
New version has been published with the changes in #17 to remedy this problem.